bindings/imap-codec-python/Cargo.toml

```diff
@@ -1,7 +1,12 @@
 [package]
 name = "imap-codec-python"
-description = "Python bindings for imap-codec"
+description = "Python bindings for imap-codec Rust crate"
+keywords = ["email", "imap", "codec", "parser"]
```

Comment:
This ...

```toml
keywords = ["email", "imap", "codec", "parser"]
```

... is also in `imap-codec/Cargo.toml`. Maybe I misunderstand, did you put this here intentionally?

Reply:
As this is the bindings package for `imap-codec`, I thought it makes sense to use the same keywords for the Python package.